lutris/gui/application.py

```diff
@@ -113,6 +115,10 @@ def add_arguments(self):
             "install", ord("i"), GLib.OptionFlags.NONE, GLib.OptionArg.STRING,
             _("Install a game from a yml file"), None,
         )
+        self.add_main_option(
```

Comment:
This shouldn't be needed, you can adapt the "install" option to detect http links